src/usr/local/www/system_advanced_notifications.php

```diff
@@ -230,93 +230,92 @@
-
-$group = new Form_Group('Secure SMTP Connection');
-$group->add(new Form_Checkbox(
+$section->addInput(new Form_Checkbox(
 	'smtpssl',
-	'Enable SSL/TLS',
+	'SSL/TLS encryption',
 	'Enable SMTP over SSL/TLS',
-	isset($pconfig['smtpssl'])
+	$pconfig['smtpssl']
 ));
 
-$section->add($group);
+$section->addInput(new Form_Input(
+	'smtptimeout',
+	'Connection timeout',
+	'number',
+	$pconfig['smtptimeout']
+))->setHelp('Number of seconds to wait for the SMTP connection to succeed. Default is 20.');
 
 $section->addInput(new Form_Input(
 	'smtpfromaddress',
-	'From e-mail address',
+	'From email address',
 	'text',
 	$pconfig['smtpfromaddress']
-))->setHelp('This is the e-mail address that will appear in the from field.');
+))->setHelp('The email address that will appear in the From field.');
 
 $section->addInput(new Form_Input(
 	'smtpnotifyemailaddress',
-	'Notification E-Mail address',
+	'To email address',
 	'text',
 	$pconfig['smtpnotifyemailaddress']
-))->setHelp('Enter the e-mail address to send email notifications to.');
+))->setHelp('The email address to send notifications to. Multiple addresses may be '.
+	'entered, separated by a comma.');
 
 // This name prevents the browser from auto-filling the field. We change it on submit
 $section->addInput(new Form_Input(
 	'smtpusername',
-	'Notification E-Mail auth username (optional)',
+	'Auth username (optional)',
 	'text',
 	$pconfig['smtpusername'],
 	['autocomplete' => 'off']
-))->setHelp('Enter the e-mail address username for SMTP authentication.');
+))->setHelp('If required by the SMTP server, enter a username for authentication.');
 
 $section->addPassword(new Form_Input(
 	'smtppassword',
-	'Notification E-Mail auth password',
+	'Auth password',
 	'password',
 	$pconfig['smtppassword']
-))->setHelp('Enter the e-mail account password for SMTP authentication.');
+))->setHelp('Password for SMTP authentication.');
 
 $section->addInput(new Form_Select(
 	'smtpauthmech',
-	'Notification E-Mail auth mechanism',
+	'Auth mechanism',
 	$pconfig['smtpauthmech'],
 	$smtp_authentication_mechanisms
-))->setHelp('Select the authentication mechanism used by the SMTP server. Most work with PLAIN, some servers like Exchange or Office365 might require LOGIN. ');
+))->setHelp('Select the authentication mechanism used by the SMTP server. Most work with PLAIN, '.
+	'some servers like Exchange or Office365 might require LOGIN.');
 
 $section->addInput(new Form_Button(
 	'test-smtp',
-	'Test SMTP Settings',
+	'Test SMTP settings',
 	null,
 	'fa-send'
-))->addClass('btn-info')->setHelp('A test notification will be sent even if the service is '.
-	'marked as disabled.  The last SAVED values will be used, not necessarily the values entered here.');
+))->addClass('btn-info')->setHelp('A test email will be sent even if the service is '.
+	'marked as disabled. If you have changed any settings, click SAVE before clicking '.
```

Comment:
I think that "If you have changed..." sentence can be removed entirely, see 64f8ec62ddd57241e4a8a755cade7ca8cd9196a6